config/routes.rb

```diff
@@ -8,5 +8,5 @@
   get "/users", to: "users#index"
   get "/users/:id/posts", to: "posts#index"
   get "/users/:id", to: "posts#show"
-  get "/users/:id/posts/:id" to: "posts#post"
+  get "/users/:id/posts/:id", to: "posts#post"
```

Comment:
[REQUIRED] Please make sure that all routes are present as you need it in the next milestone. For example:

- [ ] posts#show should be /users/:user_id/posts/:id

- [ ]  users#show  should be /users/:id